packages/wdio-spec-reporter/src/index.js

```diff
@@ -387,7 +387,7 @@ class SpecReporter extends WDIOReporter {
     getEnviromentCombo (caps, verbose = true, isMultiremote = false) {
         const device = caps.deviceName
         const browser = isMultiremote ? 'MultiremoteBrowser' : (caps.browserName || caps.browser)
-        const version = caps.version || caps.platformVersion || caps.browser_version
+        const version = caps.browserVersion || caps.browser_version || caps.version || caps.platformVersion
         const platform = caps.os ? (caps.os + ' ' + caps.os_version) : (caps.platform || caps.platformName)
 
         // Mobile capabilities
```

Comment:
Currently the version is one collection of different things, instead I suggest to instead add the following below:
If devide is true, set version to be the platform version.
Else, set the browser version.

Reply:
doen